frontend/integration-tests/protractor.conf.ts

```diff
@@ -113,6 +113,7 @@ export const config: Config = {
     return new Promise((resolve) => htmlReporter.afterLaunch(resolve.bind(this, exitCode)));
   },
   suites: {
+    baremetalSmokeTests: ['tests/metalkube/dashboard.scenario.ts'],
```

Comment:
To follow the convention which kubevirt-plugin set it would be good to call this `metal3-plugin` and colocate the metal3-plugin tests into the plugin - move the test modules into `frontend/packages/metal3-plugin/integration-tests` and reference them here as
```
metal3-plugin: ['../packages/metal3-plugin/integration-tests/tests/dashboard.scenario.ts'],
```